packages/gafl-webapp-service/src/session-cache/session-manager.js

```diff
@@ -30,6 +30,8 @@ const protectionExemptSet = [
 
 const forbiddenUnlessAgreedSet = [ORDER_COMPLETE.uri, ORDER_COMPLETE_PDF.uri, PAYMENT_FAILED.uri, PAYMENT_CANCELLED.uri]
 
+export const useSessionCookie = request => request.path.startsWith('/buy')
```

Comment:
My understanding was that we agreed to serve the session cookie for all pages which makes this redundant

Reply:
Yes - this was what I was discussing with you and Haroon on the call on Monday. Moving over to that might best best extracted as a separate ticket, I think it'll take me another day to make that change.

Reply:
I think Phil was going to do that as a separate piece of work

Reply:
👍